simphony/cuds/lattice.py

```diff
@@ -26,22 +25,25 @@ def __init__(self, index, data=None):
 
 
 class Lattice(ABCLattice):
-    """
-    A Bravais lattice;
-    stores references to data containers (node related data).
+    """A Bravais lattice
 
-    Parameters
-    ----------
-    name : str
-    type : str
+    Stores references to data containers (node related data).
+
+    Attributes:
+    -----------
+    name : string
```

Comment:
`str` is better than `string` since `str` is the name of the string type in python